language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ExtractToConstantCodeAction.java

```diff
@@ -145,6 +163,24 @@ private boolean isExpressionNode(Node node) {
         return node instanceof StatementNode || node instanceof ModuleMemberDeclarationNode;
     }
 
+    private void addRenamePopup(CodeActionContext context, CodeAction codeAction, Position replacePosition,
```

Comment:
Shall we make this method a util method as this is used in multiple code actions